client/packages/system/src/Item/api/operations.graphql

```diff
@@ -266,6 +266,29 @@ query itemById($storeId: String!, $itemId: String!) {
   }
 }
 
+fragment ItemVariantOption on ItemVariantNode {
```

Comment:
Nice, feels like a potential pattern to re-use?

Reply:
I wondered about it! In cases where we are doing a query to populate the options, this is nicer than having to map in the component - but I think many queries are reused from another list... arguably that's not great though!